app/build.gradle.kts

```diff
@@ -128,6 +128,9 @@ dependencies {
     // Compose - Testing
     androidTestImplementation(libs.compose.junit)
 
+    // Compose - Integration with ViewModels with Navigation and Hilt
+    implementation(libs.hilt.navigation.compose)
```

Comment:
We shouldn't need this import in the app. Each feature should be able to have its own ViewModel that is injected in the composable screen.